### DISCUSSION TITLE
Linker Errors for ktxTexture_* functions

### DISCUSSION THREAD

**Discussion by Owi27** [KhronosGroup/KTX-Software#878] [Q&A] [answered]
![image](https://github.com/KhronosGroup/KTX-Software/assets/69972213/8a74af8b-b597-4a6a-bd60-3ab0b30062a0)

I'm receiving linker errors using these functions and don't understand why. I'm on version 4.3.2

![image](https://github.com/KhronosGroup/KTX-Software/assets/69972213/6f723bdc-bc5e-4da2-be30-3d74c34337aa)

**Comment by MarkCallow** [comment]
It looks like you are on Windows and have either failed to include ktx.lib in your link command or are attempting to link an application compiled to use a dynamic library with a static ktx.lib.

**Reply by Owi27** [reply]
I'm such an idiot. I didn't even think of the lib🤦🏾. Thank you so much

**Reply by Owi27** [reply]
So i'm I've included the lib folder in my directories but can't seem to find ktx.lib

**Reply by MarkCallow** [reply, marked as answer]
The lib folder is source code. You have to either build the library or download the Windows installer from the project's Releases, install it and use the included ktx.lib. To build the library you can use CMake to build your project and `add_subdirectory` your KTX-Software clone in your project's `CMakeLists.txt` file.
